k256/Cargo.toml

```diff
@@ -27,7 +27,7 @@ sha3 = { version = "0.9", optional = true, default-features = false }
 criterion = "0.3"
 ecdsa-core = { version = "0.8.2", package = "ecdsa", default-features = false, features = ["dev"] }
 hex = "0.4" # TODO: switch to hex-literal
-hex-literal = "0.3"
+hex-literal = "0.2"
```

Comment:
It seems like we weren't properly testing MSRV before since we didn't have `override: true`! 😬